### ISSUE TITLE
Problem with on screen controls and mfi controllers

### ISSUE THREAD

**Issue by memoli76** [tomkidd/DOOM3-iOS#4]
On screen controls are not sufficient for example reload gun action is not present. And on screen left analog controls both movement and look around so they conflict so game is unplayable. There must be right analog also for looking around. 
     Ps4 mfi controller isnt recognized by the game

**Comment by tomkidd** [comment]
Indeed, there is no reload button (this is the only id Tech game I've ported that even had such a thing so that's an oversight on my part). I'll see about adding that. 

The ability to look around using on-screen controls can be handled by dragging on the screen or by using tilt aiming. Not necessarily ideal, and at some point I intend to add more control options. 

As for the MFi controller issue, try the latest code - there was a longstanding issue with MFi controls in tvOS, I've merged a PR that might have fixed it, though I haven't tried PS4/Xbone controllers myself.

**Comment by freefer** [comment]
Iphone 11 It cannot be controlled normally through the screen. The direction control will cause the lens to rotate rapidly. Can the direction control only keep moving up, down, left and right without rotation

**Comment by jake2244338** [comment]
> Indeed, there is no reload button (this is the only id Tech game I've ported that even had such a thing so that's an oversight on my part). I'll see about adding that.
> 
> The ability to look around using on-screen controls can be handled by dragging on the screen or by using tilt aiming. Not necessarily ideal, and at some point I intend to add more control options.
> 
> As for the MFi controller issue, try the latest code - there was a longstanding issue with MFi controls in tvOS, I've merged a PR that might have fixed it, though I haven't tried PS4/Xbone controllers myself.

What latest code
